Applications/Multimedia/Mp3tag/Online/script.js

```diff
@@ -5,8 +5,8 @@ new OnlineInstallerScript()
     .editor("Florian Heidenreich")                               
     .applicationHomepage("http://www.mp3tag.de/")  
     .author("ImperatorS79")                     
-    .url("http://download.mp3tag.de/mp3tagv283setup.exe")                       
-    .checksum("ee337d5d98b6664a74e6df87de083a9146ad422e")                       
+    .url("http://download.mp3tag.de/mp3tagv284asetup.exe")                       
```

Comment:
Is there a possibility to always get the latest version?

Reply:
I do not think it is possible